app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,24 @@
+class SessionsController < ApplicationController
+  def new
+  end
+
+  def create
+    user = User.log_in(params[:username], params[:password])
```

Comment:
Not sure if params password is necessary here bc it's encrypted- might want to double check though.